ui/src/services/api.js

```diff
@@ -54,6 +54,7 @@ export type Config = {
   flags?: [],
   url_navbar: string,
   url_navbar_config: string,
+  ui_base_path: string,
```

Comment:
should we add this `ui_base_path` in `config.json` as well?

Reply:
I added this configuration in a previous PR : https://github.com/scality/metalk8s/blob/improvement/enable-users-group-mapping-in-shell-ui-config/salt/metalk8s/addons/ui/deployed/ui.sls.in#L83

Reply:
I don't think it is necessary to be added in local config.json nor cypress one for now as if the value is null it will by default assume that it is deployed on the root path. I will make this explicit by switching this type to a nullable string through